src/Services/MediaLibrary/Glide.php

```diff
@@ -286,9 +286,6 @@ private function getOriginalMediaUrl($id)
             return null;
         }
 
-        /** @var string $endpoint */
-        $endpoint;
```

Comment:
By removing this wont we get an Undefined Variable error if `$endpointType` is not `local`, `s3` or `azure`?

Reply:
You are right!